src/T2DMap.cpp

```diff
@@ -4611,13 +4611,42 @@ void T2DMap::wheelEvent(QWheelEvent* e)
     if (e->modifiers() & Qt::ControlModifier) { // Increase rate 10-fold if control key down - it makes scrolling through
```

Comment:
Perhaps this should be 5-fold now!

Reply:
Actually 5.7: `(F^5 - 1) / F - 1)` with `F=1.07`.
I'll remove that number.